### DISCUSSION TITLE
Scheduling a Series of call in early August on Wallet Interoperabilty

### DISCUSSION THREAD

**Discussion by ChristopherA** [BlockchainCommons/Gordian-Developer-Community#64] [General & Announcements]
I've been talking to a number of members of this community about scheduling a series of calls in early August to discuss Wallet Interoperability specifications and issues. Why multiple calls? We are so spread out in time zones it is unlikely all of us can do it in one call.

Clearly one of the topics is broader deployment of UR in particular the animated PSBT QRs, but we don't need to limit ourselves to that topic alone.

If you are interested in joining a call, let me know. If there is a topic we should add to the agenda, add them to this discussion.

-- Christopher Allen

**Comment by Overtorment** [comment]
* sign arbitrary text, so a watch-only wallet that's tracking xpub could request a signature form a HW wallet

**Reply by fmishajnet** [reply]
Interested in joining such call(s), please include me.   Thank you.

**Comment by ghost** [comment]
> If you are interested in joining a call, let me know.

Interested to know different issues related to wallet interoperability. I am not sure but interoperability is not limited to UX, some of the issues affect privacy and security as well.

> If there is a topic we should add to the agenda, add them to this discussion.

1. Signing messages: Had discussed few things in https://github.com/bitcoin/bitcoin/issues/10542#issuecomment-817980713

2. Wallet fingerprinting

I understand the topic is related to privacy although different bitcoin wallets need to agree on few things and use similar implementations for lot of things to solve wallet fingerprinting.

Example: Consider there are 3 famous Bitcoin wallets used by most people A, B and C. If A doesn't allow users to spend unconfirmed UTXO, B and C are still similar for someone spying although if C uses a different locktime it breaks privacy for B as well. So A, B and C need to agree on few things to improve privacy even though there are no standards for lot of things in a Decentralized network like Bitcoin.

3. Payjoin

4. BIP 47

5. Descriptors
